extensions/shared/videopress-plan-check.scss

```diff
@@ -0,0 +1,13 @@
+.no-videopress-media-placeholder {
```

Comment:
Same question for this custom CSS, would this be better in the VideoPress block and an editor.scss file?

Reply:
In the process of refactoring this so no longer relevant.